native/c/zbase64.h

```diff
@@ -97,20 +97,10 @@ static const unsigned char *get_ebcdic_decode_table()
   return table;
 }
 
-// Fast inline function to calculate encoded size with optional padding
-inline size_t encoded_size(size_t input_size, bool with_padding = true)
+// Fast inline function to calculate encoded size
+inline size_t encoded_size(size_t input_size)
 {
-  if (with_padding)
-  {
-    return ((input_size + 2) / 3) * 4;
-  }
-  else
-  {
-    // Without padding, calculate exact size needed
-    const size_t full_blocks = input_size / 3;
-    const size_t remaining = input_size % 3;
-    return (full_blocks * 4) + (remaining > 0 ? remaining + 1 : 0);
-  }
+  return ((input_size + 2) / 3) * 4;
```

Comment:
(nitpick) C++ order of operations are P, E, MD, AS from left-right - so you can remove one set of parens:


```suggestion
  return (input_size + 2) / 3 * 4;
```

That said I'm fine with keeping it as-is if we intend to use the parens for explicit grouping/readability 😋